### DISCUSSION TITLE
File Upload Problem

### DISCUSSION THREAD

**Discussion by gctom** [livewire/livewire#8304] [7. Bugs (archived)]
### Livewire version

v3

### Laravel version

11

### Which PHP version are you using?

PHP 8.3

### Steps To Reproduce

Hi, Livewire team.

The same file upload system works on Laravel 10, I'm facing a problem with Laravel 11, the file does not upload. 

I copied all the same files from the documentation and tried to upload them but got an issue.

I created the route:
Route::any('/upload', Upload::class);

class code:
namespace App\Livewire;

use Livewire\Component;
use Livewire\WithFileUploads;
use Livewire\Attributes\Validate;
class Upload extends Component
{

    use WithFileUploads;
 
    #[Validate('image|max:1024')] // 1MB Max
    public $photo;
 
    public function save()
    {
        $this->photo->store(path: 'photos');
    }
    public function render()
    {
        return view('livewire.upload');
    }

}

and blade:
<form wire:submit="save">

        <input type="file" wire:model.defer="photo">
     
        @error('photo') <span class="error">{{ $message }}</span> @enderror
        <div wire:loading wire:target="photo">Uploading...</div>
        <button type="submit">Save photo</button>
    </form>
    
    #problem
    
    while I choose the image for upload then error message display = "The photo failed to upload." automatically without click on submit button.
    
    In console shows error
     "POST http://livewire.test/livewire/upload-file?expires=1713096082&signature=a37ba1e5f38b5310b42c43ae2f30e838afd3ab7c83f0cab308a8dedd65657dcc 500 (Internal Server Error)"
     
     If I click this console error then shows on network header:
     Request URL:
http://livewire.test/livewire/upload-file?expires=1713096082&signature=a37ba1e5f38b5310b42c43ae2f30e838afd3ab7c83f0cab308a8dedd65657dcc
Request Method:
POST
Status Code:
500 Internal Server Error
Remote Address:
127.0.0.1:80
Referrer Policy:
strict-origin-when-cross-origin

in preview tab:
{message: "Path cannot be empty", exception: "ValueError",…}
exception
: 
"ValueError"
file
: 
"C:\\laragon\\www\\livewire\\vendor\\laravel\\framework\\src\\Illuminate\\Filesystem\\FilesystemAdapter.php"
line
: 
414
message
: 
"Path cannot be empty"

In Response Tab:
  "message": "Path cannot be empty",
  
  
  
  what is the exactly problem and how can I solve this?
  
  Laravel 11, livewire 3 

### Repository or Wirebox URL

#

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by Fabian-Cabrera** [comment]
[Uncomment ](https://prnt.sc/Fhyvmaf9f1vE)upload_tmp_dir  and set value to "C:/laragon/tmp" inside [php.ini](https://prnt.sc/xxADk35i0PjK)
